Documentation/design/release.md

```diff
@@ -1,32 +1,93 @@
 # Steps to create a new release
 
-1. Bump version in OLM_VERSION file. Make a PR and wait until it has been merged.
+## Step 1: Verify Manifests
+We need to ensure that `manifests` folder is in sync with the templates in `deploy/chart/templates`.
+* Make sure you have a clean workspace. `git status` should show no change(s) or untracked file.
+* Make sure you pull the latest from `upstream`.
+* Checkout `master` branch.
+* Run `make release`
+
+This should generate no change(s) in `manifests` folder. If there are no change(s) in `manifests` folder then you can proceed with the new release. 
+You can clean your workspace 
+```bash
+git checkout -- .
+git clean -df
+```
+
+If you see change(s) in `manifests` folder, then:
+* STOP the release process
+* Consult the team and backport the changes in `manifests` folder into the templates.
```

Comment:
Whoa - have people been putting changes into the manifests without using the templates? I assumed that when this happened the templates were modified and no make release was done.

Reply:
Everything went fine for me, I just added it as a precaution. :)